ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/radius_server.schema.yml

```diff
@@ -0,0 +1,48 @@
+type: dict
+keys:
+  attribute_32_include_in_access_req:
+    type: dict
+    keys:
+      hostname:
+        type: bool
+      format:
+        type: str
+        description: Name of ACL
+  dynamic_authorisation:
+    type: dict
+    keys:
+      port:
+        type: int
+        min: 0
+        max: 65535
+        convert_types:
+        - str
+        description: Port
```

Comment:
No need for repeating the key name. Only add description if it is an explanation of what to write in the field.

Reply:
I changed the description to "TCP Port".